core/dbio/database/database.go

```diff
@@ -197,8 +197,8 @@ var (
 )
 
 func init() {
-	if os.Getenv("FILEPATH_SLUG") != "" {
-		filePathStorageSlug = os.Getenv("FILEPATH_SLUG")
+	if os.Getenv("SLING_TEMP_CLOUD_FOLDER") != "" {
```

Comment:
This new name is _much_ clearer! I imagine this commit will be a breaking change for anyone using `FILEPATH_SLUG` already. Would you want to fall back to the old env var name, at least for a few releases, while you communicate this change out?

Reply:
Thanks, I'll do that. It was never a documented env var, but just in case someone is using it.